### DISCUSSION TITLE
How to disable expansions (~/.) completions

### DISCUSSION THREAD

**Discussion by sudoAlphaX** [marlonrichert/zsh-autocomplete#747] [Q&A] [answered]
I want to disable expanding (~/.) completions. Not disable autocomplete itself, just expanding ~/. . How do i do that?

![image](https://github.com/user-attachments/assets/4bdb0756-656e-43e1-ab45-e4e080783d0d)
to
![image](https://github.com/user-attachments/assets/81637776-58ed-412d-b241-63d291bc4f8b)

Similar: #745

**Comment by michaeldebetaz** [comment]
Hey! I am facing the same issue and I cannot find any answer for the sake of me ^^ Did you find a solution?

**Reply by michaeldebetaz** [reply]
@sudoAlphaX I found a way to revert back to a working commit 🎉

Follow the steps here: https://github.com/marlonrichert/zsh-autocomplete/issues/736#issuecomment-2437351896

**Comment by polyzen** [comment, marked as answer]
Previously you could use eg. `zstyle ':completion:*' completer _complete _complete:-fuzzy _correct _approximate _ignored` to mask https://github.com/marlonrichert/zsh-autocomplete/blob/d1bbecac646cb1ab5d3a0d3b5151c42c131c0a66/Functions/Init/.autocomplete__config#L13-L14, but that no longer disables expansions after https://github.com/marlonrichert/zsh-autocomplete/commit/06431898a04cbe9d2e128a1b66b84905a8c1de69.

cc @marlonrichert

**Reply by polyzen** [reply]
Works if you make the `zstyle` call _after_ sourcing the plugin.

**Reply by L0laapk3** [reply]
Thanks! Your working config at https://github.com/polyzen/dotfiles/blob/master/base/.zshrc has helped me getting it to work too.

**Reply by michaeldebetaz** [reply]
Finally, it works for me ! It disable the expansion which is fine for me. I guess the specific solution would be to just disable the expansion for the "~".

Thank you very much.

**Comment by sudoAlphaX** [comment]
Thank you @polyzen, [that](https://github.com/polyzen/dotfiles/blob/54e02245c00c0f6bf5aee9d683bb3eac1d6c5d97/base/.zshrc#L46C1-L46C91) indeed works
